src/apps/testapps/testH3CellAreaExhaustive.c

```diff
@@ -119,7 +121,12 @@ static void cell_area_assert(H3Index cell) {
  */
 static void earth_area_test(int res, double (*cell_area)(H3Index),
                             double target, double tol) {
-    double area = mapSumAllCells_double(res, cell_area);
+    double area = 0.0;
+    IterCellsResolution iter = iterInitRes(res);
+
+    for (; iter.h; iterStepRes(&iter)) {
+        area += (*cell_area)(iter.h);
+    }
```

Comment:
Same comment here, `iter` is never used outside of the for loop, there is no purpose in declaring it outside and having a strangely formatted `for` loop.

Reply:
One man's "strange" is another man's "nice, tidy, beautiful short lines". But as you wish :)